src/api_server/swagger/firecracker.yaml

```
@@ -317,6 +317,30 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /mmds/config:
+    put:
+      summary: Set MMDS configuration.
```

Be explicit about restrictions on the API call.
```suggestion
      summary: Set MMDS configuration. Pre-boot only.
```